Bio/Align/Applications/_Clustalw.py

```diff
@@ -129,47 +129,47 @@ def __init__(self, cmd="clustalw", **kwargs):
             # ***Fast Pairwise Alignments:***
```

Comment:
This whole bunch may start with (line 45):
```python
    self.parameters = [
        _Option(...
````

Reply:
That would be cleaner, yes. And it _probably_ wouldn't mean re-indenting everything (like I had to do on one of the other application wrappers in this pull request).